modules/base/base/module.py

```diff
@@ -397,17 +397,56 @@ async def _bookmark(
         )
 
     async def _autothread(
-        self, payload: discord.RawReactionActionEvent, message: discord.Message
+        self,
+        payload: discord.RawReactionActionEvent,
+        message: discord.Message,
     ):
-        """Handle autothread functionality.
-
-        This function is not yet available on Discord publicly, nor in in
-        discord.py, so it doesn't do anything.
-        """
+        """Handle autothread functionality."""
         tc = TranslationContext(payload.guild_id, payload.user_id)
 
-        await utils.Discord.remove_reaction(message, payload.emoji, payload.member)
-        await payload.member.send(tr("_autothread", "wip", tc))
+        for reaction in message.reactions:
+            if reaction.emoji != "🧵":
+                continue
+
+            # remove if the message is threaded or is in channel with threading not enabled
```

Comment:
The comment says that the reaction is removed when threads are not enabled, but only checks if the message already has a thread.